ci-operator/jobs/konveyor/mtrho-operator/OWNERS

```diff
@@ -0,0 +1,29 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# Fetched from https://github.com/konveyor/mig-operator root OWNERS
+# If the repo had OWNERS_ALIASES then the aliases were expanded
+# Logins who are not members of 'openshift' organization were filtered out
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- alaypatel07
+- djzager
+- dymurray
+- eriknelson
+- jaydipgabani
+- jmontleon
+- pranavgaikwad
+- rayfordj
+- shawn-hurley
+- sseago
+options: {}
+reviewers:
```

Comment:
Please add @hhpatel14 and @deepakraj1997 to reviewers list

Reply:
Done!